v0/destinations/intercom/transform.js

```diff
@@ -143,6 +144,9 @@ function processSingleMessage(message, destination) {
 
   // build the response and return
   const payload = constructPayload(message, MappingConfig[category.name]);
+  if (sendAnonymousId && !payload.user_id && !payload.email) {
```

Comment:
We should only be concerned about the user ID.
```suggestion
  if (sendAnonymousId && !payload.user_id) {
```

Reply:
ok